tests/test_warp.py

```diff
@@ -264,6 +265,24 @@ def test_calculate_default_transform_multiple_resolutions():
         assert height == 20
 
 
+def test_calculate_default_transform_large_dimensions():
```

Comment:
What would you think about xfail'ing this if the system's memory is too small?

Reply:
@sgillies what does it actually tell us if it succeeds?  We have other tests that confirm that the function works properly...  And the only reason that it otherwise breaks is due to running out of memory.

I'm thinking we should drop this test unless we've determined the path forward on use of `scale`.  Please see the discussion in #1131 and give your opinions on how to proceed.  I can implement once I have an idea of appropriate direction.

Reply:
👍  on dropping the test.